src/lib.rs

```diff
@@ -101,7 +103,7 @@ pub mod server {
 
     /// Ignores incoming export name, accepts everything
     /// Export name is ignored, currently only one export is supported
```

Comment:
Does this comment stay relevant in your version of the code?

Reply:
updated